charts/osm/values.schema.json

```diff
@@ -67,6 +67,59 @@
                     "additionalProperties": false
                 }
             }
+        },
+        "autoScale": {
+            "$id": "#/properties/definitions/properties/autoScale",
+            "type": "object",
+            "title": "The autoScale schema",
+            "description": "Autoscale configuration parameters",
+            "required": [
+                "enable"
+            ],
+            "properties": {
+                "enable": {
+                    "$id": "#/properties/definitions/properties/autoScale/properties/enable",
+                    "type": "boolean",
+                    "title": "Autoscale enable",
+                    "description": "Indicates whether autoscale should be enabled or not.",
+                    "examples": [
+                        false
+                    ]
+                },
+                "minReplicas": {
+                    "$id": "#/properties/definitions/properties/autoScale/properties/minReplicas",
+                    "type": "integer",
+                    "title": "Autoscale minimum replicas",
+                    "description": "Indicates the minimum replicas for autoscale.",
+                    "minimum": 1,
+                    "maximum": 10,
```

Comment:
What if user requires more than 10 mimReplicas? I intention here is trying to avoid come back and update here in the future

Reply:
@allenlsy Does JSON schema allow using other values to define minimum/maximum? If not, such arithmetic conditions can't be enforced for 100% correctness when a combination of values are involved. If user specifies min > max, that's a user error.

Reply:
No, I don't find JSON schema allows that. My actual concern is, what if 10 replicas is not enough, and should we remove the `maximum` constraint.

Reply:
I get your point now. I don't think we need beyond 10 replicas, if we do we can update it then. There is an argument to be made to also remove the max, although it opens the door for expoits where someone can potentially overwhelm the cluster with a large number of replicas and bring it down if there is a way to constantly increase CPU load (potentially through an API endpoint that results in it).

Reply:
Ok cool. Let's use 10 as the max for now